plugins/org.eclipse.elk.alg.radial/src/org/eclipse/elk/alg/radial/options/RotationStrategy.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 
+ *******************************************************************************/
+package org.eclipse.elk.alg.radial.options;
+
+import org.eclipse.elk.alg.radial.intermediate.rotation.AngleRotation;
+import org.eclipse.elk.alg.radial.intermediate.rotation.IRadialRotator;
+
+/**
+ * The list of selectable rotation algorithms.
+ *
+ */
+public enum RotationStrategy {
```

Comment:
This seems to be a little over-engineered. Are different rotation strategies other than rotate to angle even possible? Otherwise a simple boolean would be enough.